pkg/virt-controller/watch/network_test.go

```diff
@@ -123,8 +123,114 @@ var _ = Describe("Network interface hot{un}plug", func() {
 			expectToChange,
 		),
 	)
+	DescribeTable("apply dynamic interface request on VMI",
+
+		func(vmiForVM, currentVMI, expectedVMI *v1.VirtualMachineInstance, hasOrdinalIfaces bool) {
+			vm := VirtualMachineFromVMI(currentVMI.Name, vmiForVM, true)
+			updatedVMI := applyDynamicIfaceRequestOnVMI(vm, currentVMI, hasOrdinalIfaces)
+			Expect(updatedVMI.Networks).To(Equal(expectedVMI.Spec.Networks))
+			Expect(updatedVMI.Domain.Devices.Interfaces).To(Equal(expectedVMI.Spec.Domain.Devices.Interfaces))
+		},
+		Entry("when the are not interfaces to hotplug/unplug",
+			libvmi.New(
+				libvmi.WithInterface(bridgeInterface(testNetworkName1)),
+				libvmi.WithNetwork(&v1.Network{Name: testNetworkName1}),
+			),
+			libvmi.New(
+				libvmi.WithInterface(bridgeInterface(testNetworkName1)),
+				libvmi.WithNetwork(&v1.Network{Name: testNetworkName1}),
+			),
+			libvmi.New(
+				libvmi.WithInterface(bridgeInterface(testNetworkName1)),
+				libvmi.WithNetwork(&v1.Network{Name: testNetworkName1}),
+			),
+			false),
```

Comment:
nit: consider using a const so it will read nicer.

Reply:
done.